chew.go

```diff
@@ -1,18 +1,54 @@
+/*
+Package chew provides a simple way to process URLs and files. It allows you to process a list of URLs
+and files and returns the content of the URLs and files as a list of Chunks. It also provides a way to
+transcribe audio files using the Google Cloud Speech-to-Text API or the OpenAI Whisper API.
+
+The library respects robots.txt files and crawl delays, and allows you to set a custom http.Client for making requests.
+
+Note on Responsible Usage:
+
+This library is designed for processing data from both local files and web sources. Users should be aware of the following considerations:
+
+1. Web Scraping:
+  - When scraping websites, ensure compliance with the target website's terms of service and robots.txt rules.
```

Comment:
You should add something about the fact even if the library tries to respect the robots.txt file, the user is responsible.

Or something like that